include/picongpu/param/pusher.param

```diff
@@ -62,6 +62,15 @@ namespace picongpu
         namespace sqrt_Vay = precision64Bit;
     }
 
+    namespace particlePusherHC
+    {
+        /** Precision of the square roots during the push step
```

Comment:
In your code, no sqrt operation is used directly, but this datatype is used repeatedly - please update this comment to represent the use of the high-precision data type.

Reply:
In line 95 of `include/picongpu/particles/pusher/particlePusherHigueraCary.hpp` starting with `const sqrt_HC::float_X gamma_plus` a square root is calculated. This is (one place) where we need the high precision. The other is when we calculate `mom_diff2`.

Reply:
Okay - sorry, I overlooked that `math::sqrt`